packages/sit-onyx/src/components/OnyxButton/OnyxButton.vue

```diff
@@ -89,10 +89,10 @@ const rippleEvents = computed(() => rippleRef.value?.events ?? {});
       }
 
       &.onyx-button--default {
-        --onyx-button-background-color: var(--onyx-color-base-primary-500);
-        --onyx-button-background-hover-color: var(--onyx-color-base-primary-400);
+        --onyx-button-background-color: var(--onyx-color-component-cta-default);
+        --onyx-button-background-hover-color: var(--onyx-color-component-cta-default-hover);
```

Comment:
You don't have to do it in this PR but could you talk with Jannick if its intended that the button in kaufland theme does not have a hover andripple animation color? Propably the hover color is the same as the default one

Reply:
I talked with Jannick about it and I this got changed.